adhocteam/Head-Start-TTADP
# [TTAHUB-258] Dashboard visual fixes

## Description of change

Resolved some design/visual bug feedback
- the topic/frequency graph had too much whitespace between the filters and the graph. The container wasn't staying static.
- Rename the link to the dashboard in the side navigation

Also added an error message to the date range picker if a date isn't entered in the correct format, rather than silently correcting the date to the first available in the date range

## How to test

Navigate to the dashboard and verify the changes above have been made and function appropriately.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-258

## Checklists

### Every PR


- [x] Meets issue criteria (partially)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
